docs/project/2024/maria-irina.fotescu_teo-antonio.economu/index.md

```diff
@@ -0,0 +1,77 @@
+# Kalman Temperature Alert System
+
```

Comment:
Please add a one line description of your project.

Reply:
Ok!

Reply:
Not added.

Reply:
Is it ok now?